src/components/Book.jsx

```diff
@@ -2,43 +2,57 @@ import React from 'react';
 import { CircularProgressbar } from 'react-circular-progressbar';
 import 'react-circular-progressbar/dist/styles.css';
 import PropTypes from 'prop-types';
+import Button from './Button';
 
+const btns = ['Comments', 'Remove', 'Edit'];
 function Book({
-  title, author, category, id,
+  title, author, category, id, numArr, index,
 }) {
   const categoryName = category.name;
-
   return (
     <li key={id} className="book">
       <div className="book__info">
-        <h4>{categoryName}</h4>
+        <h4>{categoryName || category}</h4>
         <h2>{title}</h2>
         <h6>{author}</h6>
         <div className="book__interaction">
-          <button type="button" className="book__btn">
-            Comments
-          </button>
-          <button type="button" className="book__btn">
-            Remove
-          </button>
-          <button type="button" className="book__btn">
-            Edit
-          </button>
+          {btns.map((btn) => (
+            <Button
+              key={btn}
+              bg="#fff"
+              font="0.875rem"
+              clr="#4386ff"
+              borderRight={`${btn}` !== 'Edit' ? '1px solid #aaa' : 'none'}
+              style={{
+                borderRight: btn !== 'Edit' ? '1px solid #aaa' : 'none',
+              }}
+              borderTRradius="0px"
+              borderBRradius="0px"
+              className="book_btn"
+              type="button"
+              content={btn}
+              padding="0.501rem 1.188rem 0.586rem 1.375rem"
+            />
+          ))}
```

Comment:
- [ ] Good add adding all the buttons required. Please note that the **remove button** an event handler to **remove** a book from the `books` array (with the *id* attribute). 

```
<Button onClick={() => dispatch(removeBook(id))}>Remove</Button>`
```